tb_plugin/torch_tb_profiler/profiler/tensor_core.py

```diff
@@ -0,0 +1,49 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# -------------------------------------------------------------------------
+class TC_Whitelist:
+    # Refer to https://github.com/NVIDIA/PyProf/blob/fd1b2902e3306119eee40ba6b6e8b2f816920c29/pyprof/prof/tc.py#L19
+    whitelist = ['h884', 's884', 'h1688', 's1688', 'hmma', 'i8816', '16816',
+                 'dgrad_1x1_stride_2x2', 'first_layer_wgrad_kernel', 'conv1x1',
+                 'conv2d_c1_k1', 'direct_group', 'xmma_implicit_gemm',
+                 'xmma_sparse_conv', 'xmma_warp_specialized_implicit_gemm',
+                 'xmma_gemm', 'xmma_sparse_gemm', 'c1688']
+
+    @classmethod
+    def is_fit(cls, item):
```

Comment:
refer to https://github.com/python/cpython/blob/main/Lib/enum.py#L609

Reply:
Thanks. Done.